src/content/app/tools/blast/components/blast-submission-header/BlastSubmissionHeader.tsx

```diff
@@ -70,19 +70,20 @@ export const BlastSubmissionHeader = (props: Props) => {
     submission.submittedData.parameters.program.toUpperCase();
   const submissionId = submission.id;
   const submissionTime = getFormattedDateTime(new Date(submission.submittedAt));
+  const { jobName } = submission.submittedData.parameters;
 
   const editSubmission = () => {
     const { sequences, species, preset, parameters } = submission.submittedData;
     const parsedSequences = sequences.flatMap((sequence) =>
       parseBlastInput(sequence.value)
     );
-    const { title, program, stype, ...otherParameters } = parameters;
+    const { jobName, program, stype, ...otherParameters } = parameters;
```

Comment:
There is a `jobName` at line number 73 too which is the same. Won't it be better to set these variables at the top itself rather than within the function?

Reply:
This shouldn't be valid anymore. Changed the code so that submissionName (jobName rename) is now separated from parameters